redash/settings.py

```diff
@@ -90,6 +90,27 @@ def all_settings():
 SAML_LOGIN_ENABLED = SAML_METADATA_URL != ""
 SAML_CALLBACK_SERVER_NAME = os.environ.get("REDASH_SAML_CALLBACK_SERVER_NAME", "")
 
+# Enables the use of an externally-provided REMOTE_USER environment var.
+# REMOTE_USER should be set to an email address
+#
+# Note that redash will only check REMOTE_USER once, upon the first need for a
+# login, and then set a cookie which keeps the user logged in.  Dropping
+# REMOTE_USER after subsequent requests won't automatically log the user out.
+# Doing so could be done with further work, but usually it's unnecessary.
+#
+# In proxy situations, it may be desirable to forward REMOTE_USER via a HTTP
+# request header, which you can specify by setting REDASH_REMOTE_USER_HEADER.
+# Doing so is *potentially dangerous*, and it is your responsibility to ensure
+# that only a trusted frontend (usually on the same server) can talk to the
+# redash backend server, otherwise people will be able to login as anyone they
+# want by directly talking to the redash backend.
+#
+# If you set REDASH_PASSWORD_LOGIN_ENABLED to false, then your authentication
+# will be seamless.  Otherwise a link will be presented on the login page to
```

Comment:
The redirect to the index page on line 15 is conditionalized on `not settings.REMOTE_USER_LOGIN_ENABLED`.  Perhaps we're using a different idea of seamless?  I mean only that the user won't be prompted to click any link to use remote user auth.  That is, under:

    settings.PASSWORD_LOGIN_ENABLED = False;
    settings.REMOTE_USER_LOGIN_ENABLED = True;

then requests needing login will redirect to `/login`, which will redirect to `/remote_user/login`, which will perform the login and redirect the client back to the original page (or the index, if no `next_page`).

Am I missing something?